components/textbook-demo/CoursePagesSection.vue

```diff
@@ -77,6 +77,12 @@ export default class CoursePagesSection extends Vue {
       segment: { action: `${this.segmentActionBase} the-atoms-of-computation` },
       url: 'https://platypus.qiskit.org/course/introduction/the-atoms-of-computation'
     },
+    {
+      image: 'the-atoms-of-computation-page-preview.png',
```

Comment:
This image doesn't seem to be related to the "What is quantum?" course.

Reply:
We don't have the final images yet :) https://github.com/Qiskit/qiskit.org/issues/1887

Reply:
Ok, got it. We'll update the image with that PR later. Thanks 🙂